src/repositories/theatre_repository.go

```diff
@@ -1,22 +1,33 @@
 package repositories
 
 import (
-	"errors"
-
 	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/.gen/KinoTicketSystem/model"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/.gen/KinoTicketSystem/table"
+	kts_errors "github.com/ELITE-Kinoticketsystem/Backend-KTS/src/errors"
 	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/managers"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/models"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/utils"
 	"github.com/google/uuid"
 )
 
 type TheaterRepoI interface {
-	GetSeatsForCinemaHall(cinemaHallId *uuid.UUID) ([]model.Seats, error)
+	GetSeatsForCinemaHall(cinemaHallId *uuid.UUID) ([]model.Seats, *models.KTSError)
 }
 
 type TheatreRepository struct {
 	DatabaseManager managers.DatabaseManagerI
 }
 
-func (tr *TheatreRepository) GetSeatsForCinemaHall(cinemaHallId *uuid.UUID) ([]model.Seats, error) {
-	// TODO: implement
-	return nil, errors.New("not implemented")
+func (tr *TheatreRepository) GetSeatsForCinemaHall(cinemaHallId *uuid.UUID) ([]model.Seats, *models.KTSError) {
+	stmt := table.Seats.SELECT(table.Seats.AllColumns).WHERE(table.Seats.CinemaHallID.EQ(utils.MysqlUuid(cinemaHallId)))
```

Comment:
From what table are you selecting? Or do you not need to specify when you select all columns from only one table?